styles/hud/cgaz.scss

```diff
@@ -18,3 +19,54 @@
 		transform: rotatez(180deg);
 	}
 }
+
+.line {
```

Comment:
Remember, classes defined in Sass files are all compiled into one `main.scss` file. So these class names feel too general, best to use `cgaz-line` etc

Reply:
re: POEditor, I was poking around with the translations, but wasn't sure about the order of operations. I'm ready to upload the English definitions when this is merged.

Reply:
Yeah I think it's best to prep them at the same time as making the PR then import once PR is merged.